operations/node_management/Add_a_Standard_Rack_Node.md

```diff
@@ -526,8 +526,7 @@ If adding Foxconn (Paradise) nodes to the system, follow the [Replacing Foxconn
 
 **These troubleshooting steps are applicable when DVS is operating over the NMN network, and not the HSN network.**
 
-There should be a `cray-cps` pod (the broker), three `cray-cps-etcd` pods and their waiter, and at least one `cray-cps-cm-pm` pod.
-Usually there are two `cray-cps-cm-pm` pods, one on `ncn-w002` and one on `ncn-w003` and other worker nodes.
+There should be one or more `cray-cps` pods.
```

Comment:
Double check with COS/DVS if this is correct

Reply:
I check with Kazuya and verified it on a running system.